docs/user/supported_devices.rst

```diff
@@ -339,6 +339,10 @@ See also: :doc:`x86`
 Footnotes
 ---------
 
+.. [#deprecated]
+  The device or target is reaching its end of life soon. This means that we cannot guarantee
```

Comment:
maybe "means that there may be not support in the next release anymore"
why? we cannot guarantee any support, and the current sentence implicitly says that we guarantee it for all but the mentioned ones...

Reply:
Updated the wording, please check again.